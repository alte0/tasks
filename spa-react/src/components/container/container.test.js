import React from 'react';
// import {  render } from '@testing-library/react';
import renderer from 'react-test-renderer';
import Container from './container';

it('Container render', () => {
    const tree = renderer
        .create(
            <Container>
                <p>text</p>
            </Container>
        )
        .toJSON();

    expect(tree).toMatchSnapshot();
});
