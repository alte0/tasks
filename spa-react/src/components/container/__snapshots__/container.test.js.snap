// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container render 1`] = `
<div
  className="container"
>
  <p>
    text
  </p>
</div>
`;
